src/components/pageComponent/login/loginForm.tsx

```diff
@@ -0,0 +1,107 @@
+import {ChangeEvent, FormEvent, useEffect, useState} from "react";
+import {useRouter} from "next/router";
+import {useSelector} from "react-redux";
+import {RootState} from "@/src/redux/index";
+
+import style from "./LoginForm.module.scss";
+import Input from "@/src/components/common/input/input";
+import Label from "@/src/components/common/input/label";
+import ButtonCta from "@/src/components/common/buttonCta/buttonCta";
+import {saveLoginInfo} from "@/src/data/data-login";
+import {loginContainerErrorMessage} from "@/src/utils/errorMessage";
+
+interface values {
+  email: string;
+  password: string;
+  confirmPassword: string;
+}
+
+interface Props {
+  signState: boolean;
+  // signState true면 signIn false면 signOut
+}
+
+function LoginForm({signState}: Props) {
+  const initialValue = {
+    email: "",
+    password: "",
+    confirmPassword: "",
+  }; // 인풋의 로그인값, 에러메세지의 초기값
```

Comment:
이런 상수값은 컴포넌트 외부에 작성해줘야합니다~
또한 타입을 정의해뒀기 때문에 이렇게 타입을 잡아주는게 좋습니다~
```suggestion
  const initialValue:InitialValue = {
    email: "",
    password: "",
    confirmPassword: "",
  }; // 인풋의 로그인값, 에러메세지의 초기값
```